Plugins/JavaCompilerPlugin/JavaCompilerPlugin.swift

```diff
@@ -32,6 +32,13 @@ struct JavaCompilerBuildToolPlugin: BuildToolPlugin {
     // so we cannot eliminate this deprecation warning.
     let sourceDir = target.directory.string
 
+    // The name of the configuration file JavaKit.config from the target for
+    // which we are generating Swift wrappers for Java classes.
+    let configFile = URL(filePath: sourceDir)
+      .appending(path: "Java2Swift.config")
+    let configData = try Data(contentsOf: configFile)
+    let config = try JSONDecoder().decode(Configuration.self, from: configData)
```

Comment:
Hmm, we probably want to make the config file optional for JavaCompiler. Let me fix this.

Reply:
makes sense yeah